packages/react/src/components/DetailsList/DetailsList.base.tsx

```diff
@@ -867,6 +867,15 @@ export class DetailsListBase extends React.Component<IDetailsListProps, IDetails
     return 0;
   }
 
+  public getCurrentColumnWidths(): { [columnKey: string]: number | undefined } {
```

Comment:
In theory, we could just define a method `getColumns(): IColumn[]` which returns `this.state.adjustedColumns`.
This would probably be more useful than a method which only returns the widths.
A caller could create a mapping of keys to widths for its own use if necessary.
We could also define a `getColumn(key: string): IColumn` method to get the full state of a specific column, too.

Reply:
@ThomasMichon Updated with `getColumns()` and removed the method that retrieves widths only

Reply:
Woops pushed another update. `this.state.adjustedColumns` doesn't actually contain the current column widths, `this._columnOverrides` does.